_plugins/helm.rb

```diff
@@ -23,15 +24,24 @@ def render(context)
 
       version = context.registers[:page]["version"]
       imageRegistry = context.registers[:page]["registry"]
-      
+
+      # Load the versions.yml file so it can be rewritten in a standard helm format.
+      versionFile = YAML::load_file('_data/versions.yml')
+      components = versionFile[version][0]["components"]
+
+      # Write the yaml values to a temp file for reading.
+      tv = Tempfile.new("temp_versions.yml")
```

Comment:
Can we delete this file after the rendering? Doesn't look like we do at the moment.

Reply:
It gets deleted when the ruby interpreter exits but it's always good to clean them up explicitly just in case. https://ruby-doc.org/stdlib-1.9.3/libdoc/tempfile/rdoc/Tempfile.html#class-Tempfile-label-Explicit+close

Reply:
Neato, that works for me.